web/src/components/DandisetsPage.vue

```diff
@@ -169,6 +180,16 @@ export default {
         data, headers, status, statusText,
       };
     },
+    async publishedVersions() {
+      if (!this.girderDandisetRequest) {
+        return null;
+      }
+      return Promise.all(this.girderDandisetRequest.data.map(
+        async (dandiset) => publishRest.mostRecentVersion(
+          dandiset.meta.dandiset.identifier, dandiset._id,
+        ),
```

Comment:
I agree it would be cleaner, but if we didn't need the girder ID, I think the destructuring would be unnecessary. I'd rather not modify the signature and overall design of the function to canonicalize a temporary hack. The more we look at this and cringe, the sooner we fix the underlying problem.